data/campaigns/Heir_To_The_Throne/overworld/00e_eastern_wesnoth.cfg

```diff
@@ -0,0 +1,1535 @@
+#                                                                    PILLAGE FARMS
+#######################################################################################################################################################
+#define PLACE_PILLAGE_FIRE X Y
+    {PLACE_FIRE {X} {Y}}
+    [terrain]
+        {AND x,y,radius={X},{Y},1}
+        {AND terrain=*^Gvs}
+        layer=overlay
+        terrain=^Edt
+    [/terrain]
+    [terrain]
+        {AND x,y,radius={X},{Y},1}
+        {AND terrain=Aa^*}
+        layer=base
+        terrain=Rb
+    [/terrain]
+#enddef
+#define PLACE_PILLAGE_FIRES
+    {PLACE_PILLAGE_FIRE 54 49} {PLACE_IMAGE "scenery/trash.png" 54 49}
+    {PLACE_PILLAGE_FIRE 52 49}
+    {PLACE_PILLAGE_FIRE 52 47} {PLACE_IMAGE "scenery/trash.png" 52 49}
+    {PLACE_PILLAGE_FIRE 51 49} {PLACE_IMAGE "scenery/trash.png" 51 49}
+    {PLACE_PILLAGE_FIRE 48 46} {PLACE_IMAGE "scenery/trash.png" 48 46}
+    {PLACE_PILLAGE_FIRE 57 42}
+    {PLACE_PILLAGE_FIRE 51 44} {PLACE_IMAGE "scenery/trash.png" 51 44}
+    {PLACE_PILLAGE_FIRE 46 46}
+    {PLACE_PILLAGE_FIRE 45 46}
+#enddef
+# this event is an in-universe way to lower the difficulty for the next scenario
+[event]
+    name=prestart
+    {IF} {VARIABLE_CONDITIONAL bm_weldyn_pillage equals yes} {THEN(
+        {PLACE_PILLAGE_FIRES}
+    )} {ELSE(
+        # put these in a location where Konrad can still access them even after Weldyn's Halberdiers wall him in
+        {SET_LABEL 53 49 _"Stockpiles"}
+        {PLACE_IMAGE "items/gold-coins-large.png"      52 49}
+        {PLACE_IMAGE "items/gold-coins-medium.png"     53 49}
+        {PLACE_IMAGE "items/gold-coins-small.png~FL()" 52 48}
+        {PLACE_IMAGE "items/gold-coins-small.png"      54 49}
+        {PLACE_IMAGE "items/gold-coins-small.png"      54 47}
+    )} {/IF}
+
+    [event]
+        name=moveto
+        first_time_only=no
+        {FILTER( side=1 {FILTER_LOCATION x,y,radius=53,49,1} )}
+        {FILTER_CONDITION({VARIABLE_CONDITIONAL bm_weldyn_pillage not_equals yes})}
+
+        #############################
+        # EXPLAIN PILLAGE
+        #############################
+        {IF} {VARIABLE_CONDITIONAL bm_weldyn_pillage equals $null} {THEN(
+            {MOVE_UNDER_KONRAD( {UNSTORE_NPC Lisar x,y=$stored_konrad.x,$stored_konrad.y side,facing=14,se} {MOVE_UNIT id=Lisar "$($stored_konrad.x+1)" $stored_konrad.y} )}
+            {MODIFY_UNIT id=Lisar facing sw}
+            [message]
+                speaker=Lisar
+                message=_"Konrad, we’re now deep within the heartland of Wesnoth. While the western farms suffer under mother’s exorbitant taxes, those surrounding Weldyn — the core of her power base — have grown rich and fat."
+            [/message]
+            [message]
+                speaker=Lisar
+                message=_"It’s an unsavory idea, but if we’re desperate, we may need to consider going scorched-earth on the locals to deprive my mother of their resources.
```

Comment:
While the concept of scorched-earth in warfare is old, I think the term itself is quite modern. I'm not sure what else to call it, though. Perhaps 'we may need to consider burning the fields to deprive my mother...'